exp/api/v1beta1/ocimanagedcluster_conversion.go

```diff
@@ -30,6 +30,9 @@ var managedclusterlogger = ctrl.Log.WithName("ocimanagedcluster-resource")
 func (src *OCIManagedCluster) ConvertTo(dstRaw conversion.Hub) error {
 	dst := dstRaw.(*v1beta2.OCIManagedCluster)
 
+	managedclusterlogger.Info("before - beta1 to beta2")
+	managedclusterlogger.Info("before - source is", "source", src)
+	managedclusterlogger.Info("before - destination is", "destination", dst)
```

Comment:
Is this mainly for testing? or will this be left in to help debug migration issues?

Reply:
I had it removed in the latest draft